lib/Template/CollaboraTemplateProvider.php

```diff
@@ -25,7 +25,11 @@ class CollaboraTemplateProvider implements ICustomTemplateProvider {
 	/** @var ITemplateManager */
 	private $coreTemplateManager;
 
-	public function __construct(TemplateManager $templateManager, IURLGenerator $urlGenerator, ITemplateManager $coreTemplateManager) {
+	public function __construct(
+		TemplateManager $templateManager,
+		IURLGenerator $urlGenerator,
+		ITemplateManager $coreTemplateManager
+	) {
```

Comment:
We can shorten the boiler plate code with https://stitcher.io/blog/constructor-promotion-in-php-8

Then you can also drop the separate propery definition above

```suggestion
	public function __construct(
		private TemplateManager $templateManager,
		private IURLGenerator $urlGenerator,
		private ITemplateManager $coreTemplateManager
	) {
	}
```